Cooper/ClientApp/src/app/components/games/games.component.html

```diff
@@ -1,22 +1,21 @@
-<h2>All games</h2>
+<h1 class="cooper-text h1--color" align="center">All games</h1>
```

Comment:
remove unnecessary class cooper-text

Reply:
Ohh, sorry)  forgot to include that file in commit )